app/controllers/registrations_controller.rb

```diff
@@ -11,8 +11,13 @@ def create
         '''
         Consume MakeSchools API to authenticate students with their database
         '''
+        if request.headers[:Cookie].nil?
+            user = MakeSchoolServer.post('https://www.makeschool.com/login.json', body: { 'user[email]' => request.headers[:email], 'user[password]' => request.headers[:password] })
+
+        else
+            user = MakeSchoolServer.post('https://www.makeschool.com/login.json', body: { 'user[email]' => request.headers[:email], 'user[password]' => request.headers[:password] }, headers: {'Cookie' => request.headers[:Cookie]})
```

Comment:
@matthewharrilal does makeschool need the password when sending a cookie?

Reply:
No, you're right. Let me get rid of that.